code/create/create-dta.sh

```diff
@@ -14,35 +11,17 @@ datalad install -d . -s git@github.com:TobiasKadelka/build_dta.git code/build_dt
 
 # modify the rules for the spec, so the dta-rules will be used
 git config -f .datalad/config --add datalad.hirni.dicom2spec.rules code/build_dta/code/create/dta_dicom2spec_rules.py
-
-# configuring the .datalad/config, so it knows where to find the procedure for renaming the fieldmaps
-# git config -f .datalad/config --add datalad.locations.dataset-procedures code/build_dta/code/procedures
-
-# THIS IS NOT NECESSARY, WHEN FILES ARE ALREADY IN TARS ON CLUSTER
-# get the source_data for DTA1
-# ./code/tmp/tar-DTA1_output.sh
-# source_data for DTA2
-# ./code/tmp/tar-DTA2_output.sh
+datalad save
```

Comment:
Should be `datalad save .datalad/config -m "Configure dataset's procedure location"`